src/main/java/org/folio/search/exception/SearchServiceException.java

```diff
@@ -1,10 +1,15 @@
 package org.folio.search.exception;
 
+import lombok.Getter;
+import org.folio.search.model.types.ErrorCode;
+
 /**
  * Thrown to indicate a service error that was occurred at operations to search engine.
  */
 public class SearchServiceException extends RuntimeException {
 
+  @Getter private final ErrorCode errorCode = ErrorCode.SERVICE_ERROR;
```

Comment:
Maybe we should introduce a base exception with the property and then all our custom exceptions have to set the error code?